Src/AutoFixture.xUnit.net.UnitTest/CompositeDataAttributeInsufficientDataTest.cs

```diff
@@ -74,17 +74,7 @@ public IEnumerator<object[]> GetEnumerator()
             yield return CreateTestCase(
                 data: new[]
                 {
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] {  new object[] { 1, 2    },  new object[] {  3,  4     }, new object[] { 5, 6 } }),
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] {  new object[] { 7, 8, 9 },  new object[] { 10, 11, 12 }                        })
-                }
-            );
-
-            yield return CreateTestCase(
-                data: new[]
-                {
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] {  1,  2     }, new object[] {  3,  4     }, new object[] { 5, 6 } }),
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] {  7,  8,  9 }, new object[] { 10, 11, 12 }                        }),
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] { 13, 14, 15 }                                                     })
+                    new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] { 1, 2, 3, 4 } }),

```

Comment:
Agreed, xUnit.net data theories throw, if the number of data rows is bigger than the number of test parameters.

Though, in the context of `CompositeDataAttribute`, if the number of data rows is bigger, why not just return a subset?

-----

In other words, given a test with 3 parameters and a data theory with 4 rows:

```
new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] { 1, 2, 3, 4 } })
```

why not just feed the test with `1`, `2`, `3` and throw `4` out? – Since we *do* have some data, why not use it...